full-service/src/json_rpc/v2/e2e_tests/webhook.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) &2020-2022 MobileCoin Inc.
+
+//! End-to-end tests for the Full Service Wallet API.
+
+#[cfg(test)]
+mod e2e_webhook {
```

Comment:
☝️ The main motivation for this change is tested here: during `account_sync`, the webhook fires that receives how many txos were received & spent in the scanned chunk